src/components/GeneralInfo/GeneralInformation/__snapshots__/GeneralInformation.test.js.snap

```diff
@@ -598,6 +598,20 @@ exports[`GeneralInformation should render correctly 1`] = `
                         >
                           Invalid date
                         </dd>
+                        <dt
+                          aria-label="Last updated title"
+                          class=""
+                          data-ouia-component-id="Last updated title"
+                        >
+                          Last updated
+                        </dt>
+                        <dd
+                          aria-label="Last updated value"
+                          class=""
+                          data-ouia-component-id="Last updated value"
+                        >
+                          Invalid date
```

Comment:
These are a clue that we either need tests, and/or we have a bug.

I'm working under the assumption that there is never a system that does not have an `entity.updated` or `entity.per_reporter_staleness` value. But, I think we can safeguard against this by displaying something other than 'Invalid date'.